### PR TITLE
prci: use f26 template for master

### PR DESCRIPTION
**Pull request by tomaskrizek**
Switch PR CI testing of master branch to Fedora 26.

Signed-off-by: Tomas Krizek <tkrizek@redhat.com>

### CODE REVIEW

**Comment by tomaskrizek** [comment]
@stlaz Please add `re-run` in the morning. It takes a while before the vagrant boxes are available.

**Comment by pvoborni** [comment]
Any idea why is there the CFFI libcrypto.so error ?

I saw the same thing when testing IPA in Vagrant with F26 box (official Fedora one).  But not in RHEVM. Both vms had openssl-libs-1.1.0f-7.fc26.x86_64 and python(2/3)-cffi-1.9.1-2.fc26.x86_64

```
2017-08-18T12:59:25Z DEBUG Created connection context.rpcclient_139893735174096
2017-08-18T12:59:25Z INFO [try 1]: Forwarding 'schema' to json server 'https://master.ipa.test/ipa/json'
2017-08-18T12:59:25Z DEBUG New HTTP connection (master.ipa.test)
2017-08-18T12:59:26Z DEBUG received Set-Cookie (<type 'list'>)'['ipa_session=MagBearerToken=DWgYzpM9CBdm%2buV7fmpPX0YvRFxwJxIS9NrILpb%2fbDahpUeJbAtlqQKOhdCGyH3k0kzl7LhQFD4IK5UQRDtBdphGMCcYpm5KiT5SFESE0rlbTZ7275eSYtrtjLxWQ8tOu16zOPNFSLvS%2fXweSb3up6cNTFTFw%2fZ5y%2bwsgqOqEUyffzIShMtzP2pohnt5Z%2bejgSJPu%2f7Q2gcEqI8LK0AUPIVlfR4eSs9pShRstUvBUdg%3d;path=/ipa;httponly;secure;']'
2017-08-18T12:59:26Z DEBUG storing cookie 'ipa_session=MagBearerToken=DWgYzpM9CBdm%2buV7fmpPX0YvRFxwJxIS9NrILpb%2fbDahpUeJbAtlqQKOhdCGyH3k0kzl7LhQFD4IK5UQRDtBdphGMCcYpm5KiT5SFESE0rlbTZ7275eSYtrtjLxWQ8tOu16zOPNFSLvS%2fXweSb3up6cNTFTFw%2fZ5y%2bwsgqOqEUyffzIShMtzP2pohnt5Z%2bejgSJPu%2f7Q2gcEqI8LK0AUPIVlfR4eSs9pShRstUvBUdg%3d;' for principal host/master.ipa.test@IPA.TEST
2017-08-18T12:59:26Z DEBUG Destroyed connection context.rpcclient_139893735174096
2017-08-18T12:59:26Z DEBUG importing all plugin modules in ipaclient.remote_plugins.schema$7419fd87...
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.remote_plugins.schema$7419fd87.plugins
2017-08-18T12:59:26Z DEBUG importing all plugin modules in ipaclient.plugins...
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.automember
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.automount
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.ca
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.cert
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.certmap
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.certprofile
2017-08-18T12:59:26Z DEBUG importing plugin module ipaclient.plugins.csrgen
2017-08-18T12:59:26Z DEBUG   File "/usr/lib/python2.7/site-packages/ipapython/admintool.py", line 174, in execute
    return_value = self.run()
  File "/usr/lib/python2.7/site-packages/ipapython/install/cli.py", line 336, in run
    cfgr.run()
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 364, in run
    self.execute()
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 388, in execute
    for _nothing in self._executor():
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 430, in __runner
    exc_handler(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 459, in _handle_execute_exception
    self._handle_exception(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 449, in _handle_exception
    six.reraise(*exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 420, in __runner
    step()
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 417, in <lambda>
    step = lambda: next(self.__gen)
  File "/usr/lib/python2.7/site-packages/ipapython/install/util.py", line 81, in run_generator_with_yield_from
    six.reraise(*exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/util.py", line 59, in run_generator_with_yield_from
    value = gen.send(prev_value)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 654, in _configure
    next(executor)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 430, in __runner
    exc_handler(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 459, in _handle_execute_exception
    self._handle_exception(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 517, in _handle_exception
    self.__parent._handle_exception(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 449, in _handle_exception
    six.reraise(*exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 514, in _handle_exception
    super(ComponentBase, self)._handle_exception(exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 449, in _handle_exception
    six.reraise(*exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 420, in __runner
    step()
  File "/usr/lib/python2.7/site-packages/ipapython/install/core.py", line 417, in <lambda>
    step = lambda: next(self.__gen)
  File "/usr/lib/python2.7/site-packages/ipapython/install/util.py", line 81, in run_generator_with_yield_from
    six.reraise(*exc_info)
  File "/usr/lib/python2.7/site-packages/ipapython/install/util.py", line 59, in run_generator_with_yield_from
    value = gen.send(prev_value)
  File "/usr/lib/python2.7/site-packages/ipapython/install/common.py", line 66, in _install
    for _nothing in self._installer(self.parent):
  File "/usr/lib/python2.7/site-packages/ipaclient/install/client.py", line 3626, in main
    install(self)
  File "/usr/lib/python2.7/site-packages/ipaclient/install/client.py", line 2346, in install
    _install(options)
  File "/usr/lib/python2.7/site-packages/ipaclient/install/client.py", line 2688, in _install
    api.finalize()
  File "/usr/lib/python2.7/site-packages/ipalib/plugable.py", line 733, in finalize
    self.__do_if_not_done('load_plugins')
  File "/usr/lib/python2.7/site-packages/ipalib/plugable.py", line 425, in __do_if_not_done
    getattr(self, name)()
  File "/usr/lib/python2.7/site-packages/ipalib/plugable.py", line 614, in load_plugins
    self.add_package(package)
  File "/usr/lib/python2.7/site-packages/ipalib/plugable.py", line 641, in add_package
    module = importlib.import_module(name)
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/lib/python2.7/site-packages/ipaclient/plugins/csrgen.py", line 25, in <module>
    from ipaclient import csrgen_ffi
  File "/usr/lib/python2.7/site-packages/ipaclient/csrgen_ffi.py", line 128, in <module>
    sk_num = _libcrypto.sk_num
  File "/usr/lib64/python2.7/site-packages/cffi/api.py", line 837, in __getattr__
    make_accessor(name)
  File "/usr/lib64/python2.7/site-packages/cffi/api.py", line 833, in make_accessor
    accessors[name](name)
  File "/usr/lib64/python2.7/site-packages/cffi/api.py", line 778, in accessor_function
    raise AttributeError('%s: %s' % (name, e))

2017-08-18T12:59:26Z DEBUG The ipa-client-install command failed, exception: AttributeError: sk_num: "function 'sk_num' not found in library 'libcrypto.so.1.1': /lib64/libcrypto.so.1.1: undefined symbol: sk_num"
2017-08-18T12:59:26Z ERROR sk_num: "function 'sk_num' not found in library 'libcrypto.so.1.1': /lib64/libcrypto.so.1.1: undefined symbol: sk_num"
2017-08-18T12:59:26Z ERROR The ipa-client-install command failed. See /var/log/ipaclient-install.log for more information
```

**Comment by pvoborni** [comment]
I have my answer: https://pagure.io/freeipa/issue/7110

File: .freeipa-pr-ci.yaml [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{fedora-25/build_url}'
+        build_url: '{fedora-26/build_url}'
         test_suite: test_integration/test_caless.py::TestServerReplicaCALessToCAFull
-        template: *ci-master-f25
+        template: *ci-master-f26
+        timeout: 3600
```
Comment (stlaz) [inline .freeipa-pr-ci.yaml]: Should this PR be pushed with this timeout, the timeout addition should be explained and added in another commit.

Comment (stlaz) [reply .freeipa-pr-ci.yaml]: Never mind, the timeout is there for the other jobs as well, we can push this.

**Comment by stlaz** [comment]
master:

* d88718cadc947a24e194d2c071d9565498614504 prci: use f26 template for master
